spec/lib/vmdb/config_spec.rb

```diff
@@ -14,7 +14,7 @@
     it "normal" do
       resource = FactoryGirl.create(:miq_server)
       MiqRegion.seed
-      data = Settings.to_hash
+      data = {}
```

Comment:
this puts everything into a known state

since it passes, this looks like it picked a good "known state"